src/segmantic/seg/monai_unet.py

```diff
@@ -137,14 +161,15 @@ def default_augmentation(self, keys: List[str]):
             xforms.append(
                 RandZoomd(keys=keys, prob=0.2, min_zoom=0.8, max_zoom=1.3, mode=mode)
             )
-
+        # ToDo: Don't hardcode ratios. Maybe pass it in config file as well?
         xforms += [
             RandCropByLabelClassesd(
                 keys=keys,
                 label_key="label",
                 spatial_size=self.spatial_size,
                 num_classes=self.num_classes,
-                num_samples=4,
+                num_samples=self.num_samples,
+                ratios=[0, 1, 2, 1, 1, 1, 1, 1, 1, 2, 1, 3, 1, 4, 3, 2, 2, 2, 2, 2]
```

Comment:
instead of the hard-coded ratios you could define the transforms in the `config.yml` (or `config.json`) using the "bundle" approach introduced [here](https://github.com/dyollb/segmantic/issues/34)

Reply:
Example config.yml that specifies the augmentation under `augmentation`:
```yml
augment_intensity: true
augment_spatial: true
augmentation:
  _disabled_: true
  _target_: Compose
  transforms:
  - _disabled_: false
    _target_: RandAdjustContrastd
    gamma:
    - 0.5
    - 4.5
    keys: '@image_key'
    prob: 0.2
  - _disabled_: false
    _target_: RandGibbsNoised
    alpha:
    - 0.0
    - 1.0
    keys: '@image_key'
    prob: 0.2
  - _target_: RandKSpaceSpikeNoised
    global_prob: 0.1
    keys: '@image_key'
    prob: 0.2
  - _target_: RandRotated
    keys:
    - '@image_key'
    - '@label_key'
    mode:
    - bilinear
    - nearest
    prob: 0.2
    range_x: 0.4
  - _target_: RandRotated
    keys:
    - '@image_key'
    - '@label_key'
    mode:
    - bilinear
    - nearest
    prob: 0.2
    range_y: 0.4
  - _target_: RandZoomd
    keys:
    - '@image_key'
    - '@label_key'
    max_zoom: 1.3
    min_zoom: 0.8
    mode:
    - area
    - nearest
    prob: 0.2
  - _target_: SpatialPad
    spatial_size:
    - 96
    - 96
    - 96
  - _target_: RandCropByLabelClassesd
    image_key: '@image_key'
    keys:
    - '@image_key'
    - '@label_key'
    label_key: '@label_key'
    num_classes: 2
    num_samples: 4
    spatial_size:
    - 96
    - 96
    - 96
cache_rate: 1.0
checkpoint_file: C:/temp/hippocampus/epoch=94-val_loss=0.07-val_dice=0.8734.ckpt
dataset: C:/test/Dataset.json
gpu_ids:
- 0
max_epochs: 100
mixed_precision: true
num_channels: 1
output_dir: C:/test/junk
spatial_dims: 3
spatial_size: null
tissue_list: C:/test/label/tissue_list.txt
```